index.html

```diff
@@ -1,12 +1,107 @@
 <!DOCTYPE html>
-<html lang="en">
+<!-- newly added 
+    * add height 100%
+    * reset padding 0
+    * reset margin 0
+-->
+<html lang="en" style="height: 100%; padding: 0; margin: 0;">
```

Comment:
The style="..." conent in the html element is not necessary. Please remove it @pisethchhom

Reply:
the style height: 100% make the page full screen and footer appear in the bottom if I remove it the footer will not always appear in the bottom